docs/proposals/mvp-dashboard-design.md

```diff
@@ -43,18 +43,125 @@ can query the relevant test builds.
 
 ## Proposal
 
-![Diagram 1](../images/perf-dash-proposal-diagram.png)
+![alt text](../images/perf-dash-proposal-diagram.png "Perf-Dash Diagram")
+
+### Server
+
+The system we propose requires the development of a server API that will be
+used to serve queries that a user makes via the dashboard. To speed up the
+initial development process, these queries will be hardcoded for test results.
+Adding query build functionality for the user is a feature that will be
+developed in future development cycles.
+
+When the server receives input from the user (a query), the server will send
+these queries to the data storage.  The data storage returns the filtered
+performance test results. These results are then pushed to the dashboard.
+The server expects a json result from the data store. The format and structure
+of this json response is a task that will need to be completed during the
+development of the perf-dash.
+
+### Dashboard
+
+The dashboard serves the purpose of allowing the user to make queries of the
+results stored in the data storage unit. The dashboard provides a simple user
+interface with a list of drop down menus for options such as: lab, config,
+test, run, and metric. The dashboard communicates with the server (both
+running on the same node) using an NGINX reverse proxy.
+
+The query is made via HTML requests in the form of `GET /api/labs`,
+`GET /api/labs/{lab id}`. The expected result is a json with the appropriate
+test data. This json is then used to create the graph of the specified metrics
+as they trend over past builds.
 
 ## Rationale
 
-A discussion of alternate approaches and the trade offs, advantages, and
-disadvantages of the specified approach.
+### Choice to build our own front-end
+
+1. **Avoid Grafana learning curve**
+
+   [Grafana][4] is used to facilitate performance monitoring in the performance
+   test lab. The team is familiar with it and we have templated various
+   visualizations for tracking the relevant metrics from each node in our
+   cloud. Using Grafana still requires time to use, time that other teams do
+   not have to invest. Our goal with the perf-dash is to allow other teams to
```

Comment:
Following off of @meteorfox 's point, I dont agree that this is the rationale, but I wanted to add that we should probably make a section for choice of making our own API vs using Influx DB API